app/js/toDoToggle.js

```diff
@@ -3,20 +3,20 @@ const openProject = () => {
   projectButtons.forEach(projectButton => {
     projectButton.addEventListener("click", e => {
       const projectIconBody = document.querySelector(
-        ".c-container--project-body"
+        ".c-container__project-body"
       );
       projectTarget = e.target.getAttribute("data-id");
       iconBorders = [...document.querySelectorAll(".c-projects__icon--border")];
       iconBorders.forEach(iconBorder => {
         if (projectTarget === iconBorder.getAttribute("data-id")) {
-          iconBorder.classList.add("borderActive");
+          iconBorder.classList.add("c-projects__icon--border--active");
         }
       });
       renderToDoList(projectTarget);
-      const projectToDo = document.querySelector(".c-container--project-list");
-      projectIconBody.classList.add("activeProject");
-      projectToDo.classList.add("activeList");
-      e.target.classList.add("iconActive");
+      const projectToDo = document.querySelector(".c-container__project-list");
+      projectIconBody.classList.add("c-container__project-body--left-justify");
```

Comment:
Does `project-body` need to be an element on `c-container`? Can't it be its own thing?